src/maptm/src/maptm.h

```diff
@@ -39,6 +39,10 @@
 #include "osp_ps.h"
 
 #define MAPTM_MODULE_NAME           "MAPTM"
+#define MAPTM_MODULE                "MAPTM"
```

Comment:
```suggestion
#define MAPTM_MODULE_NAME           "MAPTM"
```

IINM, you're using `MAPTM_MODULE_NAME` for `Node_Config` and `MAPTM_MODULE` for `Node_State`.
I'd say `MAPTM_MODULE_NAME` should be used in both cases.

Reply:
MAPTM_MODULE is used for both Node_State and Node_Config.

Reply:
@MalekSST, please clarify:
* What is `MAPTM_MODULE_NAME` used for.
* What is `MAPTM_MODULE` used for.

Reply:
MAPTM_MODULE_NAME: refers to the module name (used in strcmp() when we need to compare the module name to another string".
MAPTM_MODULE: refers to the module itself (used in ovsdb_where_simple_typed()).
The intention behind defining these macros was providing more clearance. But if it resulted more confusing, then using a single one can be better.

Reply:
Well, as it is, if you change one but not the other, things won't work as designed. So, yes, please use just `MAPTM_MODULE_NAME`, as suggested.